src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.product-card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  line-height: 14px;
+  font-weight: 400;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &--catalog {
+    margin-right: $card-margin-right;
+    margin-bottom: 46px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+  }
```

Comment:
Strange solution, create element (`catalog__product-card`) for that purpose instead of modifiers